wikipron/config.py

```diff
@@ -35,6 +36,8 @@
 _PHONEMES_REGEX = r"/(.+?)/"
 _PHONES_REGEX = r"\[(.+?)\]"
 
+_TONES_REGEX = r"[˥˦˧˨˩⁰¹²³⁴⁵⁶⁷⁸⁹⁻⁽⁾◌̋ ◌̌ ◌̏ ◌̀ ◌́ ◌̂ ◌̄ ◌᷄◌᷅◌᷆◌᷇◌᷈◌᷉]"
```

Comment:
This looks relatively complete except that I don't see the markers for downstep/upstep or for "global rise" and "global fall". Don't know if those are actually present in any of the transcripts but might be wise to include.
![Screenshot 2020-07-22 at 10 43 50 AM](https://user-images.githubusercontent.com/159289/88190620-7dc3a580-cc08-11ea-9ea5-891319f18378.png)